.github/workflows/dotnetcore.yml

```diff
@@ -47,61 +37,14 @@ jobs:
         name: webapp
         path: ${{ github.workspace }}/web
 
-    - name: Upload functional test build artifacts back to GitHub
-      uses: actions/upload-artifact@v3
-      with:
-        name: functionaltests
-        path: ${{ github.workspace }}/functionalTests
     - name: Upload probot build artifacts back to GitHub
       uses: actions/upload-artifact@v3
       with:
         name: probot
         path: src/ProbotMetrics
 
 
-  test:
```

Comment:
Please add this back. The tests are an important step to ensure that everything is working as intended.